pkg/ebpftracer/c/headers/common/network.h

```diff
@@ -122,6 +124,8 @@ typedef struct net_event_context {
     u8 argnum;
     struct { // event arguments (needs packing), use anonymous struct to ...
         u8 index0;
+        u8 direction;// 0 = flow_unknown, 1 = flow_incoming, 2 = flow_outgoing
```

Comment:
Is there a reason why you can't do `flow_direction direction` instead?

Reply:
Enum sizes are compiler dependent, hence going with `u8` is the safer bet here, haha.

Reply:
Can you explicitly specify enum sizes?

Reply:
No, this is only possible starting with C21.